src/crypto/elliptic_common.cpp

```diff
@@ -231,12 +229,11 @@ namespace fc { namespace ecc {
 
     static std::string _to_base58( const extended_key_data& key )
     {
-        size_t buf_len = key.size() + 4;
-        char *buffer = (char*)alloca(buf_len);
+        char buffer[key.size() + 4]; // it's a small static array => allocate on stack
         memcpy( buffer, key.begin(), key.size() );
         fc::sha256 double_hash = fc::sha256::hash( fc::sha256::hash( key.begin(), key.size() ));
         memcpy( buffer + key.size(), double_hash.data(), 4 );
-        return fc::to_base58( buffer, buf_len );
+        return fc::to_base58( buffer, key.size() );
```

Comment:
This looks wrong. `buf_len` used to be equal to `key.size()+4`.

Reply:
Sure, it should be the local buffer size. Poor me, such a mistake.

__Thanks!__